sdk/src/main/assets/hcaptcha-form.html

```diff
@@ -71,7 +71,7 @@
         if (renderConfig.size === 'invisible') {
             hcaptcha.execute(hCaptchaID);
         } else {
-            BridgeObject.onLoaded();
+            BridgeObject.onOpen();
```

Comment:
Maybe we should add a few comments here to explain what's actually going on in terms of control flow / bridge notification.

Reply:
@e271828- @DSergiu there are several comments about the same topic so I should like to explain my vision here

1. Before this PR we have `BridgeObject.onLoaded` method. And I think that it's method name doesn't correspond it's  semantic. i.e. we called `BridgeObject.onLoaded` directly on `open-callback`, which have nothing in common with 'loading'
2. We can keep both `onLoaded` and `onOpen` separately. But if we call 'native' methods we always have overhead on each such call, I didn't measured it but two native calls will definitely take much resources/time than one

Open question:
1. should we fire `open-callback` for case `renderConfig.size !== 'invisible'` (code above). I'm not sure but It seems like we should

Reply:
> - onLoaded is triggered when the js api is loaded
>    - We have this already

I think you are talking about function which we are passing as `onload` param to `https://js.hcaptcha.com/1/api.js`. This PR don't change this logic and we continue use [`onHcaptchaLoaded` function](https://github.com/hCaptcha/hcaptcha-android-sdk/blob/main/sdk/src/main/assets/hcaptcha-form.html#L140)

This is also my point to rename/replace `BridgeObject .onLoaded` to avoid this confusion

> - open-callback is triggered when a challenge is shown to the user.
>   - We need a new cb BridgeObject.onOpen to be triggered on open-callback in [code here](https://github.com/hCaptcha/hcaptcha-android-sdk/blob/main/sdk/src/main/assets/hcaptcha-form.html#L135). It currently triggers onLoaded but it should be the following...

Totally agree with this, but to be on the same page should we trigger for `passive` scenario (when captcha don't show)?

Reply:
> Totally agree with this, but to be on the same page should we trigger for passive scenario (when captcha don't show)?

IMO we should not, we shall expose similar behavior to what the js api does which will not trigger `open-cb` in passive mode. From the ticket itself: "we want to let apps capture the open-callback event" which sounds to me just a proxy implementation.

Reply:
> IMO we should not, we shall expose similar behavior to what the js api does which will not trigger open-cb in passive mode.

Got it

> You are right. This specific line should be reverted back to BridgeObject.onLoaded(); and moved to function onHcaptchaLoaded.

But this PR doesn't move this call out of `onHcaptchaLoaded`. `function onHcaptchaLoaded` stays untouched. But if we decided to move it out of `function execute` to `function onHcaptchaLoaded` then we will need to add it to `function resetAndExecute` also

My point what that function name `BridgeObject.onLoaded()` (and corresponding method `HCaptchaJSInterface.onLoaded` method on java side) is confusing. It actually doesn't related to any 'loading'. And I think it's make sens to rename it

----

I will put back `onLoaded`, and keep it separately from `onOpen` callback